src/main/java/com/work/management/logging/LogResponseAdviceAdapter.java

```diff
@@ -0,0 +1,42 @@
+package com.work.management.logging;
+
+import com.work.management.service.log.LoggingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.MethodParameter;
+import org.springframework.http.MediaType;
+import org.springframework.http.server.ServerHttpRequest;
+import org.springframework.http.server.ServerHttpResponse;
+import org.springframework.http.server.ServletServerHttpRequest;
+import org.springframework.http.server.ServletServerHttpResponse;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyAdvice;
+
+@ControllerAdvice
+final class LogResponseAdviceAdapter implements ResponseBodyAdvice {
+
+  private final LoggingService loggingService;
+
+  @Autowired
+  public LogResponseAdviceAdapter(LoggingService loggingService) {
+    this.loggingService = loggingService;
```

Comment:
You can even remove public from the Constructor...so that nobody outside logging package can make instance of this class by calling constructor, and he is forced to use your instance only.(through dependency injection).